src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dailyPrice = 40;
+  const bigDiscount = 50;
+  const smallDiscount = 20;
+  let result = dailyPrice * days;
+
+  if (days >= 7) {
+    result = days * dailyPrice - bigDiscount;
```

Comment:
and return value here, without reassigning result value

Reply:
Oh, like that?
```suggestion
   return result = days * dailyPrice - bigDiscount;
```

Reply:
I think i understood wrong, i'll try another thing.

Reply:
```suggestion
    return days * dailyPrice - smallDiscount;
```
or something like this

Reply:
like if we have a constant we cant change it, but we have to in order to make correct result.